### PR TITLE
libgbinder: update to 1.1.28.

### PR DESCRIPTION
**Pull request by JamiKettunen**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/libgbinder/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'libgbinder'
 pkgname=libgbinder
-version=1.1.26
+version=1.1.27
```
Comment (classabbyamp) [inline srcpkgs/libgbinder/template:3]: 1.1.28 available

Comment (JamiKettunen) [reply srcpkgs/libgbinder/template]: Updated

Comment (classabbyamp) [reply srcpkgs/libgbinder/template]: and now .29 is available 🙃

Comment (JamiKettunen) [reply srcpkgs/libgbinder/template]: I guess this is better than the projects that release no new tags for 2 years lol, updated to .29 and tested as working (with Waydroid)

**Pull request merged**